homework/style.css

```diff
@@ -1,3 +1,179 @@
+* {
+  box-sizing: border-box;
+  font-family: Roboto;
+}
+
+#root {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
 .alert-error {
-  color: red;
+  color: #ffffff;
+  background-color: #b00020;
+  width: 100%;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  padding-left: 25px;
+  margin-top: 10px;
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.main-header {
+  width: 70%;
+  background-color: #5e35b1;
+  color: white;
+  font-weight: bold;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  padding-left: 25px;
+}
+
+#main {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 70%;
+}
+
+section {
+  width: 49%;
+  margin-top: 5px;
+}
+
+.header-text {
+  float: left;
+}
+
+.repo-select {
+  margin-left: 20px;
+  width: 200px;
+  border-radius: 3px;
+}
+
+.repo-select:focus {
+  border-color: #ffffff;
+}
+
+.contributions-section {
+  box-shadow: 0px 0px 4px #7e57c2;
+  border-radius: 3px;
+}
+
+table {
+  box-shadow: 0px 0px 4px #7e57c2;
+  padding-left: 25px;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  width: 100%;
+  margin-bottom: 5px;
+  line-height: 1.5em;
+  border-radius: 3px;
+}
+
+table:hover {
+  box-shadow: 0px 0px 8px #311b92;
+}
+
+.td-header {
+  font-weight: bold;
+  margin-left: 30px;
+  width: 12%;
```

Comment:
![image](https://user-images.githubusercontent.com/2788771/65543137-d6ef7980-df10-11e9-8bed-35ec98076430.png)

It would look a bit better for long texts (e.g. Description) if the label was vertically aligned to the top.

```css
vertical-align: top;
```

![image](https://user-images.githubusercontent.com/2788771/65543173-eec6fd80-df10-11e9-9e39-fb092830c5e0.png)

Reply:
you are right it looks better this way. I implemented the change, but I will push it with the week3 homework.